src/App.tsx

```diff
@@ -21,14 +39,15 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length === 0
```

Comment:
will be awesome to handle loader with separate variable
for now if you receive an empty array from server, user will see infinite loader

Reply:
(not important, just suggestion)

Reply:
done